monai/transforms/croppad/dictionary.py

```diff
@@ -103,24 +107,69 @@
     "RandCropByLabelClassesDict",
 ]
 
-PadModeSequence = Union[Sequence[Union[NumpyPadMode, PytorchPadMode, str]], NumpyPadMode, PytorchPadMode, str]
 DEFAULT_POST_FIX = PostFix.meta()
```

Comment:
this is not needed any more ?

Reply:
OK, removed in the latest commit.

Thanks.